### PR TITLE
[Test] Avoid code scanning falsely identify

### LINKED ISSUE #3195: [Bug]: Placeholder keys cause false alarms on code-scanning tools
### Describe the bug

The following lines cause automated code-scanning tools to falsely identify the presence of an API key in the code.

https://github.com/microsoft/autogen/blob/7ceee5dadce85633b9545580cd0e4d765a9ad3e9/test/oai/test_utils.py#L51C4-L58C7

Revise the placeholder to make more obviously fake. E.g., 

```
    {
        "model": "gpt-35-turbo-v0301",
        "tags": ["gpt-3.5-turbo", "gpt35_turbo"],
        "api_key": "11111222223333344444555556666677",
        "base_url": "https://123456.openai.azure.com",
        "api_type": "azure",
        "api_version": "2024-02-15-preview"
    },
````

### Steps to reproduce

_No response_

### Model Used

_No response_

### Expected Behavior

_No response_

### Screenshots and logs

_No response_

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by wenngong**
Fix #3195

## Checks

- [x] I've included any doc changes needed for https://microsoft.github.io/autogen/. See https://microsoft.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [x] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [x] I've made sure all auto checks have passed.

### CODE REVIEW

File: test/oai/test_utils.py [hunk 1]
```diff
@@ -55,7 +55,7 @@
     {
         "model": "gpt-35-turbo-v0301",
         "tags": ["gpt-3.5-turbo", "gpt35_turbo"],
-        "api_key": "111113fc7e8a46419bfac511bb301111",
+        "api_key": "keyhere",
         "base_url": "https://1111.openai.azure.com",
         "api_type": "azure",
         "api_version": "2024-02-01"
```
Comment (Hk669) [inline test/oai/test_utils.py:61]:
```suggestion
        "api_key": "azureoaiapikey",
        "base_url": "https://deployment_name.openai.azure.com",
        "api_type": "azure",
        "api_version": "2024-02-01"
```

**Review by Hk669 — COMMENTED** [review]
this looks more verbose. @afourney please let us know your thoughts on this.

**Comment by wenngong** [comment]
GitGuardian Security also check history commits? After I add new commit, security still failed at old commit id, why?

**Comment by wenngong** [comment]
Since GitGuardian also scan the history commit, I will close this PR and submit a new clean one.

**Pull request closed without merging**